### ISSUE TITLE
[Suggestion]

### ISSUE THREAD

**Issue by flavienbonvin** [budgie-desktop/budgie-desktop#1141]
OS: Solus

It could be great to be able to choose on which corner (or the exact location) of a panel in dock mode, allowing a bit more flexibility with this nice feature

**Comment by gabrc52** [comment]
Cool! You mean being able to align the dock to the left or right apart from the right?

**Comment by flavienbonvin** [comment]
@gabrc52 Yep, actually you only can choose the side of the panel, it could be great to be able to chose the position on this side, like bottom left, bottom right.

With the actual version of Budgie the dock automatically go to the center of the side and the size change depending on the number of items you have in it, this is a bit limiting i find
